### DISCUSSION TITLE
Will 2.0 require Java 11 or Java 17?

### DISCUSSION THREAD

**Discussion by reftel** [resilience4j/resilience4j#1812] [Q&A] [answered]
In earlier discussions, it has been said that 2.0 will be based on Java 11 ( e.g. #1676 ). In the release notes for 2.0.0, however, it says "Upgraded to Java 17". Does that mean that you´re using Java 17 to compile, but with Java 11 as a target, or will Java 17 be required in applications that use the new release?

**Comment by RobWin** [comment, marked as answer]
Hi,
yes, Java 17 LTS is also almost a year old now. And Java 11 LTS almost 4 years old.
And my goal would also be to target Java17 like Spring 6 or Spring Boot 3 instead of Java 11.
But what is your opinion on this? 

I think it's about time to move one.

**Reply by reftel** [reply]
Thank you for the very quick and clear reply!
For us, dropping support for Java 11 will (for boring, big-org internal technical reasons) take quite a while. That means we´d be stuck on Resilience4j 1.7.1 until then, which would be unfortunate.
